msal-java-articles/advanced/managed-identity.md

```diff
@@ -40,7 +40,7 @@ If your application already uses one of the SDKs, continue using the same SDK. U
 To quickly get started and see Azure Managed Identity in action, you can use one of the samples the team built for this purpose:
 
 > [!div class="nextstepaction"]
```

Comment:
Not sure if it's just an issue with viewing it in Github, but this bit of script doesn't seem load correctly (you just see `[!div class="nextstepaction"]` before the link to the sample)

Since it's just a single link to our one sample you could probably get rid of it entirely.

Reply:
This won't render in GitHub, but will render OK on `learn.microsoft.com` :)